### DISCUSSION TITLE
Does Spotless plugin support Hocon file type formatting

### DISCUSSION THREAD

**Discussion by sashankchittoory** [diffplug/spotless#1980] [Q&A]
Hi there,

I'm currently using the Spotless plugin for code formatting in my project, and I'm wondering if it supports Hocon file type code style checks/formatting. I couldn't find any specific information about Hocon files in the documentation or discussions.

Could someone please clarify if Spotless can be configured to handle Hocon files for code formatting?

More at : https://github.com/lightbend/config/blob/main/HOCON.md#hocon-human-optimized-config-object-notation

**Comment by nedtwigg** [comment]
I don't think we have HOCON support, but happy to take a PR for it:

- https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md#how-to-add-a-new-formatterstep
